Common/AnnoParser.hs

```diff
@@ -237,7 +238,8 @@ floatingAnno ps = literal2idsAnno ps Float_anno
 
 prefixAnno :: Range -> GenParser Char st Annotation
 prefixAnno ps = do
-    prefixes <- many $ do
+    prefixes <- many $ skipMany (commentLine >> endOfLine) >> do
```

Comment:
Thanks for suggesting `spaces`. It worked and also enabled me to get rid of a line that I wanted to be gone from the get go. See 868229b below.
The reason I'm skipping the comments is that `prefixAnno` is already parsing a `%prefix( ... )%` annotation block and it needs to return a [`Prefix_anno`](https://github.com/spechub/Hets/blob/b259e3b3e05433b2018f45d5781000fa3af7cbdc/Common/AS_Annotation.der.hs#L108). Since I wasn't able to figure out where to put the `Unparsed_anno` returned from `commentLine` I decided to skip those comments.